src/DotNetWorker.Grpc/DotNetWorker.Grpc.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFrameworks>net5.0;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net5.0;net6.0;net7.0;netstandard2.0</TargetFrameworks>
```

Comment:
Can you detail the need for the additional TFMs? Is this to leverage the new API to retrieve the host/executable handle? This may require additional discussion.

Reply:
Synced offline. Collectively agreed to keep this new TFMs.